R/pkg/R/functions.R

```diff
@@ -3200,19 +3212,27 @@ setMethod("lag",
 #' This is equivalent to the \code{LEAD} function in SQL.
 #'
 #' @param x Column to compute on
-#' @param offset Number of rows to offset
+#' @param offset Number of rows to offset. If not specified, the default is 1.
```

Comment:
in @rdname lag, there's a bit more information on these @param. Perhaps here with @rdname lead we could do similar?